testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/arquillian/CockroachdbContainerProvider.java

```diff
@@ -1,81 +0,0 @@
-/*
- * Copyright 2023 Red Hat, Inc. and/or its affiliates
- * and other contributors as indicated by the @author tags.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package org.keycloak.testsuite.arquillian;
-
-import java.lang.annotation.Annotation;
-import org.jboss.arquillian.container.spi.event.StartSuiteContainers;
-import org.jboss.arquillian.core.api.annotation.Observes;
-import org.jboss.arquillian.test.api.ArquillianResource;
-import org.jboss.arquillian.test.spi.enricher.resource.ResourceProvider;
-import org.jboss.arquillian.test.spi.event.suite.AfterSuite;
-import org.jboss.logging.Logger;
-import org.testcontainers.containers.CockroachContainer;
-import org.testcontainers.utility.DockerImageName;
-
-public class CockroachdbContainerProvider implements ResourceProvider {
-
-    private static final Logger log = Logger.getLogger(CockroachdbContainerProvider.class);
-
-    private static final Boolean START_CONTAINER = Boolean.valueOf(System.getProperty("cockroachdb.start-container", "false"));
```

Comment:
We still mention some of these properties in `HOW_TO_RUN.md`. Do we want to remove it here? Or in a separate PR?